examples/thalamus_sample_dir/LICENSE.txt

```diff
@@ -34,4 +34,43 @@ Original URL: http://senselab.med.yale.edu/ModelDB/ShowModel.cshtml?model=3670&f
 Author: Etay Hay, Shaul Druckmann, Srikanth Ramaswamy, James King, Werner Van Geit, Christian Roessert
 Original URL: https://senselab.med.yale.edu/ModelDB/ShowModel.cshtml?model=139653&file=\L5bPCmodelsEH\mod\
 
+6) TC_Nap_Et2, persistent sodium current
+Authors: Etay Hay, Shaul Druckmann, Srikanth Ramaswamy, James King, Werner Van Geit, Elisabetta Iavarone
+Accession: 139653
+Original URL: https://senselab.med.yale.edu/modeldb/ShowModel.cshtml?model=139653&file=/L5bPCmodelsEH/mod/Nap_Et2.mod#tabs-2
+
+7) SK_E2, calcium-activated potassium current
+Authors: Etay Hay, Shaul Druckmann, Srikanth Ramaswamy, James King, Werner Van Geit
+Accession: 139653
+Original URL: https://senselab.med.yale.edu/modeldb/ShowModel.cshtml?model=139653&file=/L5bPCmodelsEH/mod/SK_E2.mod#tabs-2
+
+8) Intracellular calcium dynamics
+Authors: Etay Hay, Shaul Druckmann, Srikanth Ramaswamy, James King, Werner Van Geit
+Accession: 139653
+Original URL: https://senselab.med.yale.edu/modeldb/ShowModel.cshtml?model=139653&file=/L5bPCmodelsEH/mod/CaDynamics_E2.mod#tabs-2
+
```

Comment:
It looks like 7) is already present in 5).

Also 6) and 8) link to the same URL as in 5). I think we could add them to the list under 5) and remove items 6 to 8

Reply:
done in the next commit